### PR TITLE
feat(digest): skip digest when referencing a sidecar from the current sha-1 commit 

### PR DESCRIPTION
**Pull request by benoitf**
### What does this PR do?
skip digest when referencing a sidecar from the current sha-1 commit 

### What issues does this PR fix or reference?
PR are failing when modifying sidecar folder

### How to test this PR?
see if build will be fine as it includes a copyright change in sidecar folder

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [ ] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [ ] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [ ] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: sidecars/asciidoc/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2020 Red Hat, Inc.
+# Copyright (c) 2020-2021 Red Hat, Inc.
```
Comment (ericwill) [inline sidecars/asciidoc/Dockerfile:1]: I assume this is a test change?

Comment (benoitf) [reply sidecars/asciidoc/Dockerfile:1]: yes once I have something that works I will remove it 👍

Comment (ericwill) [reply sidecars/asciidoc/Dockerfile]: :+1:

Comment (benoitf) [reply sidecars/asciidoc/Dockerfile]: ok it works :-)

**Comment by benoitf** [comment]
@ericwill so if there is a change in the current commit on sidecar folder, it's skipped. (it doesn't work if it's separate commits)

**Comment by ericwill** [comment]
> @ericwill so if there is a change in the current commit on sidecar folder, it's skipped. (it doesn't work if it's separate commits)

Sounds fine to me, currently it's documented that sidecar PRs should be separate, so we'll just enforce that for now.

**Review by ericwill — APPROVED** [review]

**Pull request merged**